app/config/AppConfig.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class AppConfig @Inject()(servicesConfig: ServicesConfig) {
+
+  lazy val desBaseUrl: String = servicesConfig.baseUrl("des")
+
+  lazy val environment: String = servicesConfig.getString("microservice.services.des.environment")
+  lazy val authorisationToken: String = servicesConfig.getString("microservice.services.des.authorisation-token")
```

Comment:
Can you use `config.get[String]` instead to match the lines specified in the other backends?
https://github.com/hmrc/income-tax-cis/blob/main/app/config/AppConfig.scala#L33
https://github.com/hmrc/income-tax-expenses/blob/main/app/config/AppConfig.scala#L30
https://github.com/hmrc/income-tax-employment/blob/main/app/config/AppConfig.scala#L55

Reply:
I cannot use config.get since the config is not passed to the AppConfig. This was a duplication. If anything this should be changed in the other modules. It does not make sense for us to use both of the following, when only one is enough: config: Configuration, servicesConfig: ServicesConfig

Reply:
OK. This seems to be the case for `income-tax-submission-frontend`. I'll mark this conservation as resolved.